honeybread-mgmtadmin/src/main/java/com/whatsub/honeybread/mgmtadmin/domain/store/StoreService.java

```diff
@@ -0,0 +1,85 @@
+package com.whatsub.honeybread.mgmtadmin.domain.store;
+
+import com.whatsub.honeybread.common.util.Sha256Utils;
+import com.whatsub.honeybread.core.domain.base.BaseEntity;
+import com.whatsub.honeybread.core.domain.category.CategoryRepository;
+import com.whatsub.honeybread.core.domain.store.Store;
+import com.whatsub.honeybread.core.domain.store.StoreRepository;
+import com.whatsub.honeybread.core.domain.user.UserRepository;
+import com.whatsub.honeybread.core.infra.errors.ErrorCode;
+import com.whatsub.honeybread.core.infra.exception.HoneyBreadException;
+import com.whatsub.honeybread.mgmtadmin.domain.store.dto.StoreCreateRequest;
+import com.whatsub.honeybread.mgmtadmin.domain.store.dto.StoreUpdateRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor
+@Service
+@Transactional(readOnly = true)
+public class StoreService {
+
+    private final StoreRepository storeRepository;
+    private final UserRepository userRepository;
+    private final CategoryRepository categoryRepository;
+
+    @Transactional
+    public Long create(final StoreCreateRequest storeCreateRequest) {
+        checkSellerIdExistence(storeCreateRequest.getSellerId());
+        checkStoreNameExistence(storeCreateRequest.getBasic().getName());
+        checkCategoryIdsExistence(storeCreateRequest.getCategoryIds());
+
+        Store saved = storeRepository.save(storeCreateRequest.toEntity());
+        saved.updateUuid(Sha256Utils.generate(saved.getId(), saved.getBasic().getName()));
+        return saved.getId();
+    }
+
+    @Transactional
+    public void update(final long storeId, final StoreUpdateRequest storeUpdateRequest) {
+        Store savedStore = getSavedStore(storeId);
+        checkStoreNameExistenceExcludeSelf(savedStore, storeUpdateRequest.getBasic().getName());
+        checkCategoryIdsExistence(storeUpdateRequest.getCategoryIds());
+
+        savedStore.update(storeUpdateRequest.toEntity());
+    }
+
+    private Store getSavedStore(final long storeId) {
+        return storeRepository.findById(storeId).orElseThrow(() -> {
+            throw new HoneyBreadException(ErrorCode.STORE_NOT_FOUND);
+        });
+    }
```

Comment:
별건 아닌데 이렇게 변경하는건 어떨까요?
```suggestion
    private Store getSavedStore(final long storeId) {
        return storeRepository.findById(storeId)
            .orElseThrow(() -> new HoneyBreadException(ErrorCode.STORE_NOT_FOUND));
    }
```

Reply:
넘넘 좋슴댱
비슷한 맥락으로 테스트코드 throw exception 부분도 가독성 좋도록 수정해줬어요~
감사해요 ㅇㄲㄲ~~